docker-compose.yml

```diff
@@ -72,18 +72,12 @@ services:
     env_file:
       - .env
       - docker/minio/.env
-    # Create empty buckets on every container startup
-    # Note: $0 is included in the exec because "/bin/bash -c" swallows the first
-    # argument, so it must be re-added at the beginning of the exec call
```

Comment:
This comment needs to be moved to `s3_entrypoint.sh` as well rather than be deleted.

Reply:
@arun-chib this comment is still missing from the file these actions were moved to.

Reply:
Done @AetherUnbound 

> One comment needs to be moved, then this should be ready!

Done @AetherUnbound